Packs/ContentManagement/Scripts/CustomPackInstaller/CustomPackInstaller.py

```diff
@@ -65,6 +65,9 @@ def main():
             )
         )
 
+        if installation_status != 'Success':
```

Comment:
do we have a reason for the failure?
where can the user get more information about it?